mantidproject/mantid
# New ISIS indirect diffraction reduction algorithm

Fixes [#10855](http://trac.mantidproject.org/mantid/ticket/10855).

To test:
- See that system tests are passing
- Ensure both Indirect Diffraction and Data Reduction (Energy Transfer) interfaces still work
  - Run ISIS Calibration with IRS26173
  - Run ISIS Energy Transfer with IRS26173 and calibration workspace, plot contour enabled
  - Should get a nice, straight energy transfer over all Q
  - Run Indirect Diffraction with IRS26176, diffspec
- Review documentation changes.
